### PR TITLE
Discussion: One class rep

### PR DESCRIPTION
**Pull request by marcotuts**
NOTE: I'm not sure if we have a process for pull requests, but I figured this way we can start a conversation. If we decide to not merge in the proposal we can close this and open a new PR later. 

DESCRIPTION: coming soon.

### CODE REVIEW

File: oac_bylaws.markdown [hunk 1]
```diff
@@ -22,8 +22,7 @@ Any person who has been enrolled in a full-time degree program at Olin College i
 
 Section II: Composition 
 -----------------------
-1. The OAC is composed of two representatives from each graduating class in addition to an Executive Board.
-1. The OAC must ensure that each class has a Class Gift Captain to interface with the College on matters pertaining to the Class Gift. This role is not a voting member of the OAC. This role may be filled by an elected Class Representative or by another member of the OAA.
+1. The OAC is composed of one Class Representative from each graduating class in addition to an Executive Board. This representative serves as Class Gift Captain to interface with the College on matters pertaining to alumni donations.
```
Comment (marcotuts) [inline oac_bylaws.markdown]: modification --> revert this change / section.

**Comment by gregmarra** [comment]
These amendments were approved at the March 8th, 2015 meeting of the OAC.

**Comment by gregmarra** [comment]
This amendment was approved at the March 8th, 2015 OAC meeting.
